### DISCUSSION TITLE
NPMPlus and Unraid

### DISCUSSION THREAD

**Discussion by Tumtum2814** [ZoeyVid/NPMplus#884] [Q&A] [answered]
Currently using stock NPM on Unraid. Want to integrate crowdsec, so figured NPMPlus looks awesome over the lepresidente NPM container. Using NPMPlus in Unraid, originally installed it on my custom NPM network to use docker networking. Reading the readme i see its suggested to use in Host, but Unraid uses 80 and 443 for the host machine. NPMplus on my custom network started properly and i grabbed my TLS cert, it worked for a few of the AARs but couldn't reach emby  (can still reach it locally). Wondering if it has something to do with the network mode. What are the impacts from using NPMplus not in host mode?

**Comment by Zoey2936** [comment, marked as answer]
see https://github.com/ZoeyVid/NPMplus/discussions/691

**Reply by Tumtum2814** [reply]
appreciate the link! i don't think i fall into any of those buckets but i still can't seem to get NPMplus functioning in a custom network (not using crowdsec yet).
I know I'm the problem here, is there a better forum to ask for help? 
All of my Proxy hosts that i try to create give me connection refused when I'm on a custom network, but the ports are mapped. 
![image](https://github.com/ZoeyVid/NPMplus/assets/24609930/46b3dff0-ddad-424c-bf73-ffda66ca84c0)

**Reply by Zoey2936** [reply]
can you sent me your envs?

**Reply by Tumtum2814** [reply]
I actually woke up and with a fresh brain, realized the problem is my router. It wasn't updating the tables properly to change where my external 443 was routing. I wiped the nvram and it created the updated tables. Working now. Sorry to bother.

**Comment by Revan335** [comment]
We have a Guide for a Unraid Installation?

**Reply by Zoey2936** [reply]
no

**Comment by Kopernikus1979** [comment]
@Zoey2936 
Hi, also Unraid here, I run my dockers on their own VLAN, so  each ip wil can use 80/443 for example (also better isolation).
At the moment just classic NPM would NPM Plus work in my case?

**Reply by Zoey2936** [reply]
NPMplus should work, but as I said, there are no docs for unraid
